### PR TITLE
Feature newpull request

### PR DESCRIPTION
**Pull request by kinginthenorthcodez**
#1 
-latest pull request for contact form changes
-Added form to use  Formspree service.
-collaboration @HectorTorresE

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -96,186 +82,206 @@ <h4 class="work-card-title">
...
 
-           <label for="message">
-             <textarea name="textarea" id="message" placeholder="Enter text here" maxlength="300" required > 
-             </textarea>
-             </label>
+      <label for="email">
+        <input type="email" id="email" name="email" placeholder="Email address" required>
+      </label>
 
-           <button type="button" class="btn-form">Get in touch</button>
```
Comment (micheaol) [inline index.html:243]: Please ensure that the form is submitted successfully after clicking the submit `<button>`. Please check the `<button>` `type`, you might want to change it from `<button>` to `<submit>` I think this is the reason the APP is not performing as expected.

File: index.html [hunk 2]
```diff
@@ -96,186 +82,206 @@ <h4 class="work-card-title">
...
-                            </a>
-                        </li>
-                        <li>
-                            <a href="#"> <img src="assets/images/Icons/angellist.svg" alt="angellist-icon">
-                            </a>
-                        </li>
-                    </ul>
-                </div>
+      <label for="message">
+        <textarea name="textarea" id="message" placeholder="Enter your message here" maxlength="500" rows="5" required>Enter your message here
```
Comment (micheaol) [inline index.html:246]: Please ensure you stick to the design templates in Figma as much as possible, this includes the font, colors, images, and spacing (margin/padding).

## Your design: 
your-design

## Figma design:
figma

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @kinginthenorthcodez 🥇 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 


## Required Changes ♻️
- _Check the comments under the review._
- Please ensure you customize the README.md file to your project

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (kinginthenorthcodez) [reply index.html:246]: made changes as per review for the margins and fonts

**Comment by kinginthenorthcodez** [comment]
Updated changes as required 👍🏽

**Review by rotimiazeez — APPROVED** [review]
# Hi Team👏, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🎉
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kinginthenorthcodez** [comment]
> # Hi Team👏,
> ### Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 🎉
> Congratulations! 🎉
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hi @rotimiazeez  i see in the dashboard it still showing submit, thanks for the reviews.

**Pull request merged**